node/pkg/watchers/evm/connectors/batch_poller.go

```diff
@@ -101,6 +101,7 @@ func (b *BatchPollConnector) SubscribeForBlocks(ctx context.Context, errC chan e
 				sub.unsubDone <- struct{}{}
 				return nil
 			case v := <-innerErrSink:
+				b.logger.Info("batch_poller innerErrSink received an error, forwarding it on", zap.String("v", v))
```

Comment:
Error or Info?

Reply:
No need to log two errors for the same event, so I made this one an info. It's just to verify that the event made it this far.